.grazie.en.yaml

```diff
@@ -0,0 +1,27 @@
+# The "en" in the file name specifies the natural language to be checked, so users can use other ISO codes, e.g. "ru" or "de"
```

Comment:
Change "e.g" to "for example"

Reply:
Thanks, I fixed this. Apparently the highlighting works in AsciiDoc, but not in comments. 

Please see this PR more as an invitation to discuss if this feature is useful.